force-app/main/default/lwc/ldsNotifyRecordUpdateAvailable/ldsNotifyRecordUpdateAvailable.js

```diff
@@ -0,0 +1,55 @@
+import { LightningElement, api, wire } from 'lwc';
+import updateContact from '@salesforce/apex/ContactController.updateContact';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import {
+    getRecord,
+    getFieldValue,
+    notifyRecordUpdateAvailable
+} from 'lightning/uiRecordApi';
+
+import FIRSTNAME_FIELD from '@salesforce/schema/Contact.FirstName';
+import LASTNAME_FIELD from '@salesforce/schema/Contact.LastName';
+
+const fields = [FIRSTNAME_FIELD, LASTNAME_FIELD];
+
+export default class LdsNotifyRecordUpdateAvailable extends LightningElement {
+    @api recordId;
+
+    @wire(getRecord, { recordId: '$recordId', fields })
+    contact;
+
+    get firstName() {
+        return getFieldValue(this.contact.data, FIRSTNAME_FIELD);
+    }
+
+    get lastName() {
+        return getFieldValue(this.contact.data, LASTNAME_FIELD);
+    }
+
+    handleContactUpdate() {
+        updateContact({
```

Comment:
Just a thought: instead of writing our own `updateContact` Apex method, could we simply use the `updateRecord` from the UI API so that we have less code and fewer tests or does it defaut the use of `notifyRecordUpdateAvailable`?

If yes, we probably want to leave a comment to explain that in general we shouldn't write a simple update Apex method but we need it for this example.

Reply:
Added the comment